.travis.yml

```diff
@@ -41,6 +41,10 @@ install:
       fi
   - pip install -U tox twine wheel codecov
 
+addons:
+  hosts:
+    - xn--pxaix.localhost
```

Comment:
what is the problem with mocking `socket.gethostbyname` aside of @kmike not liking mocking :) ?